api/server/router/network/network_routes.go

```diff
@@ -70,6 +71,8 @@ SKIP:
 		// info is available for network specific GET API (equivalent to inspect)
 		if versions.LessThan(httputils.VersionFromContext(ctx), "1.28") {
 			nr = n.buildDetailedNetworkResources(nw, false)
+		} else if netFilters.Include("dangling") {
+			nr = n.buildDetailedNetworkResources(nw, true)
```

Comment:
Is it a concern that when filtering for "dangling", the returned results will be different than without? (i.e. it having "detailed" information)?

Looking at `buildDetailedNetworkResources()`, and how we use that information during filtering (we're only interested in knowing _if_ containers or services are attached); https://github.com/moby/moby/blob/be146652108f7145e902c275807f9ef71464b031/api/server/router/network/network_routes.go#L359-L403


Wondering if we should have a simplified version, or at least reset the `Containers` and `Services` information afterwards, so that the information that's returned for the network is consistent.